_includes/payment-order-other-features.md

```diff
@@ -279,16 +283,36 @@ for future purchases. The payer is, by default, asked if they want to store
 their payment details, so even with `generatePaymentToken` set to `true`, it is
 still up to the payer if they want the details stored or not.
 
-{% include alert.html type="informative" icon="info" body="Please note that not
-all payment instruments provided by Swedbank Pay support Payer Awareness today."
+{% include alert.html type="informative" icon="info" body="Remember that the
```

Comment:
Let's add a `header` to this alert as well? Saying something like "GDPR Responsibility".